File: Dataset/Leetcode/train/94/94.java
 class Solution {
    List<Integer> res = new ArrayList<Integer>();
    public List<Integer> XXX(TreeNode root) {
        dfs(root);
        return res;
    }
    private void dfs(TreeNode node){
        if(node == null) return;
        dfs(node.left);
        res.add(node.val);
        dfs(node.right);
    }
}

undefined
for (i = 0; i < document.getElementsByTagName("code").length; i++) {     console.log(document.getElementsByTagName("code")[i].innerText); }

File: Dataset/Leetcode/train/70/94.cpp
class Solution {
public:
    int XXX(int n) {
        int n1 = 1, n2 = 1;
        for(int i = 1; i < n ; i++) {
            n2 = n1 + n2;
            n1 = n2 - n1;
        } 
        return n2;
    }
};

